index.html

```diff
@@ -56,98 +56,106 @@ <h2 class="connect"> <a href="#">Let's connect</a></h2>
     <section class="works">
       <article class="card card-1">
         <img src="./images/tonic1.png" alt="Tonic" class="project-image">
-
-        <div class="card-headers">
-          <h2><a href="#" class="project-title">Tonic</a></h2>
-          <div class="card-categories">
-            <a href="#" class="main-category">Canopy</a>
-            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
-            <a href="#" class="sec-category">Back End Dev</a>
-            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
-            <a href="#" class="sec-category">2015</a>
+        <div class="card-content">
```

Comment:
- [ ] Since I can't find the `ul` with the class `menu` online 21 I am adding a request here. For your `li` (Portfolio, About, Contact) add a margin in between them and also make them clickable